Twig/ProfilerExtension.php

```diff
@@ -108,4 +109,43 @@ public function getData()
             'strategyClass' => $this->strategyClass,
         ];
     }
+
+    /**
+     * String representation of object.
+     *
+     * @link http://php.net/manual/en/serializable.serialize.php
+     *
+     * @return string the string representation of the object or null
+     *
+     * @since 5.1.0
+     */
+    public function serialize()
+    {
+        return serialize($this->getData());
+    }
+
+    /**
+     * Constructs the object.
+     *
+     * @link http://php.net/manual/en/serializable.unserialize.php
+     *
+     * @param string $serialized <p>
+     *                           The string representation of the object.
+     *                           </p>
```

Comment:
Please remove the HTML description and leave the text only (so without the `<p>` tags)